src/main/java/uk/gov/digital/ho/pttg/application/util/MaxLengthNameNormalizer.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.digital.ho.pttg.application.util;
+
+import uk.gov.digital.ho.pttg.dto.Individual;
+
+public class MaxLengthNameNormalizer implements NameNormalizer {
```

Comment:
There is an awful lot of code here where we could have a very simple solution - name = name.substring(0,MAX_NAME_LENGTH).  @SWhitlam Is this an occasion where you would advocate simplicity over an elaborate solution?